### DISCUSSION TITLE
setup prices as one-to-one relation but edit "inline"

### DISCUSSION THREAD

**Discussion by hcarsten** [pimcore/pimcore#11928] [Q&A / Support]
We have pretty complex product database built in PimCore already. The data is mainly provided with a connector to an SAP ERP. When the record in the SAP changes we would do a version in PimCore. The problem is in 99,99% of the case only the prices changes, so going from normal to discounted and back to normal. Somethings this could happen 2 or 3 times a day. Creating new version would can result to new version record, which when produce 50K or more new versions every day !
The most obvious solution might be to first to shift the prices to a second entity and suppress the version-generation here.
But our main problem this to have the to shown as inline fields in the product editors. How this could be done.

So basically we would need to have for the "product-class" versioning been intact, but for a related "product-prices-class" the version is turned off. But then we would like a user to be able to edit the prices inline (as a tab in the product-row) ?

best
